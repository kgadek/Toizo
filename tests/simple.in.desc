Prosty test, względem wejścia tylko jedna żarówka powinna być obrócona
