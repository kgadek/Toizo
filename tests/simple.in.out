3 1
I1 H1 I3
